servicetalk-tcp-netty-internal/src/main/java/io/servicetalk/tcp/netty/internal/TcpServerBinder.java

```diff
@@ -131,7 +132,8 @@ protected void initChannel(Channel channel) {
                             // of connection processing.
                             defer(() -> connectionAcceptor.accept(conn).concat(succeeded(conn)))
                                     // subscribeOn is required to offload calls to connectionAcceptor#accept
-                                    .subscribeOn(executionContext.executor()));
+                                    .subscribeOn(executionContext.executor(),
+                                            connectionAcceptor.requiredOffloads()::hasOffloads));
```

Comment:
`hasOffloads` -> using this method in instances where we are applying targeted offloading makes extensibility of interface limited. For example if we add another influencer method later (e.g. for closing, etc.) then this case will start offloading when it shouldn't have to. Seems like we should use `isConnectOffloaded` here.